mmaction/datasets/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .audio_visual_dataset import AudioVisualDataset
 from .ava_dataset import AVADataset
 from .base import BaseDataset
+from .blending_utils import CutmixBlending, MixupBlending  # noqa
```

Comment:
remove noqa

Reply:
And these should also be added in `__all__`